includes/sdk-migration/sdk-browser-2-migration/user-agent-parsing.md

```diff
@@ -0,0 +1 @@
+In Browser 2.0, we removes functionality to parse user agent that populates `event.os_name`, `event.os_version`, `event.device_model`, `event.device_manufacturer` on the client side. This is replaced with user agent server-side parsing which offered by Amplitude ingestion endpoints. This breaking changes might affect your chart analystic related to the user agent properties. However, we provided a [user agnet enrichment plugin](https://www.npmjs.com/package/@amplitude/plugin-user-agent-enrichment-browser) to enrich the user agent info to keep it backward compatible. Please check [here](https://github.com/amplitude/Amplitude-TypeScript/tree/v1.x/packages/plugin-user-agent-enrichment-browser) for more details.
```

Comment:
```suggestion
Starting Browser SDK 2.0, Amplitude replaced enrichment of user properties relating to user agent from client-side to server-side. The enriched user properties include `os_name`, `os_version`, `device_model`, `device_manufacturer`. While the new enrichment strategy yields more accurate results, it also yields slightly different results than Browser SDK 1.0 and may impact your existing analytics charts that query these properties. To prevent this breaking change from impacting you, install [@amplitude/plugin-user-agent-enrichment-browser](https://www.npmjs.com/package/@amplitude/plugin-user-agent-enrichment-browser) to configure Amplitude to enrich these user properties on the client-side and yield enrichment results similar to Browser SDK 1.0. Please visit our [documentation](https://www.npmjs.com/package/@amplitude/plugin-user-agent-enrichment-browser) for more details.
```

Reply:
Can we also add code samples so customers don't have to jump from one page to another?

Reply:
Any doc or analytics we can point to proving the new strategy is more accurate and there is only a slight difference between the results? 
@kevinpagtakhan

Reply:
I thought we have reviewed an internal doc that compares these two strategies in the past. We had convictions that server-side user agent parsing is what we want hence this is what we are using for Browser SDK 2.0. We can sync up offline if you have any concerns.